From 8d57b88611573a8c0b11c94a3fb207c9d65579a5 Mon Sep 17 00:00:00 2001
From: Sam Nazarko <email@samnazarko.co.uk>
Date: Sun, 11 Dec 2016 04:44:02 +0000
Subject: [PATCH] -45dBm is a good signal; we should not block with a scan
 which is expensive. Further, WLAN drivers cannot be trusted to accurately
 report signal. This prevents bgscan from thinking we are roaming away from
 the base station

Signed-off-by: Sam Nazarko <email@samnazarko.co.uk>
---
 plugins/wifi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/wifi.c b/plugins/wifi.c
index dfe849f..bba1cb2 100644
--- a/plugins/wifi.c
+++ b/plugins/wifi.c
@@ -63,7 +63,7 @@
 #define INACTIVE_TIMEOUT  12	/* in seconds */
 #define FAVORITE_MAXIMUM_RETRIES 2
 
-#define BGSCAN_DEFAULT "simple:30:-45:300"
+#define BGSCAN_DEFAULT "simple:30:-65:300"
 #define AUTOSCAN_DEFAULT "exponential:3:300"
 
 #define P2P_FIND_TIMEOUT 30
-- 
2.7.4

